doc/nrf/app_dev/config_and_build/hardware/use_gpio_pin_directly.rst

```diff
@@ -7,21 +7,36 @@ Driving a GPIO pin directly
    :local:
    :depth: 2
 
-This user guide shows how to drive a GPIO pin directly.
-In the development phase of an embedded project, it can be used to collect execution timings or as a debugging tool.
+This user guide shows how to control a GPIO pin directly using GPIO driver APIs.
+In the development phase of an embedded project, you can use this configuration to set the pin state using software, without any intermediary circuitry or additional hardware components.
+You can use this solution to collect execution timings, or as a debugging tool.
 
-Declaring the GPIO pin
-**********************
+The following configuration can be copied manually into the devicetree overlay files.
```

Comment:
```suggestion
You can copy the following configuration manually into the devicetree overlay files.
```
What is exactly the following configuration? I do not see one right below.

Reply:
I assume the configuration snippets. This remained from the v1 of the page that did not have the numbered sections.